.github/ISSUE_TEMPLATE/highlighting_issue.yml

```diff
@@ -0,0 +1,100 @@
+name: Highlighting issue
+description: Missing or incorrect highlights or you want to change the way something is highlighted
```

Comment:
probably can leave out "or you want..." as that is arguably already covered by "incorrect"?

Reply:
we can also discuss personal preferences here :slightly_smiling_face: There's not always a clear indications what's the best option and sometimes some discussion is required.

Reply:
I see what you mean -- maybe "Syntax highlighting for a specific language that is missing, incorrect, or can/should be improved"?